src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -264,7 +265,10 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
             if (_state == AzureState::TermConnected)
             {
                 // Close the websocket connection
-                _cloudShellSocket.close();
+                std::ignore = WinHttpWebSocketClose(_webSocket.get(), WINHTTP_WEB_SOCKET_SUCCESS_CLOSE_STATUS, nullptr, 0); // throw away the error

```

Comment:
Why not log it?

Reply:
Nothing useful to learn. We're going down and releasing all the resources either way. If it doesn't close, well, that sucks, but winhttp gets to deal with it when we call CloseHandle